src/app/forms/suggestion-box/terra-suggestion-box.component.ts

```diff
@@ -419,4 +407,47 @@ export class TerraSuggestionBoxComponent implements OnInit, OnChanges
     {
         this._selectedValue = value;
     }
+
+    private changeSelectedSuggestionBoxEntryBySearchString(searchString:string):void
+    {
+        // suggestion box entry we use when we do not find a suggestion box value which matches our search string
+        let fallbackSuggestionBoxEntry:TerraSuggestionBoxValueInterface = TerraSuggestionBoxHelper.generateSuggestionBoxEntryFromCaption(
+            searchString);
+
+        // suggestion box entry which we want to use as new selected entry
+        let suggestionBoxEntry:TerraSuggestionBoxValueInterface = TerraSuggestionBoxHelper.getSuggestionBoxEntryForCaption(searchString,
+            this._displayListBoxValues, fallbackSuggestionBoxEntry);
+
+        this.changeSelectedSuggestionBoxEntry(suggestionBoxEntry);
+    }
+
+    private changeSelectedSuggestionBoxEntry(suggestionBoxEntry:TerraSuggestionBoxValueInterface):void
+    {
+        // do nothing if input provides no meaningful information
+        if(isNullOrUndefined(suggestionBoxEntry))
+        {
+            return;
+        }
+
+        // update selected value
+        this._selectedValue = {
+            caption: suggestionBoxEntry.caption,
+            value:   suggestionBoxEntry.value
+        };
+
+        // update last selected values
+        if(this.inputWithRecentlyUsed)
+        {
+            this.updateLastSelectedValues();
+        }
+
+        // update temp selected value
+        this._tmpSelectedValue = this._selectedValue;
+
+        // execute callback functions
+        this.onTouchedCallback();
+        this.onChangeCallback(suggestionBoxEntry.value);
+        this.outputValueChanged.emit(suggestionBoxEntry);
```

Comment:
Brauch man diesen EventEmitter dann wirklich noch. Möchte man bei jeder Änderung in der Eingabe benachrichtig werden und darauf reagieren??